cmd/flags.go

```diff
@@ -92,6 +100,14 @@ func (f *algoFlag) Allowed() string {
 	return strings.Join(out, ", ")
 }
 
+func getKeys(m map[string]tpmutil.Handle) []string {
+	keys := make([]string, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+	return keys
+}
+
```

Comment:
We also no longer need this helper function.

We actually do this "get the keys from a map" operation twice in `gotpm`. Unfortunately, it's with a `map[string]tpmutil.Handle` and a `map[string][]tpm2.HandleType`. As Go lacks generics, we can use a helper function (we would need two helper functions.

Reply:
done